pkg/controller/util.go

```diff
@@ -362,10 +363,23 @@ func MakeCloneSourcePodSpec(image, verbose, pullPolicy, pvcName string, generate
 			GenerateName: podName,
 			Annotations: map[string]string{
 				AnnCloningCreatedBy: "yes",
+				AnnTargetPodNamespace: pvc.Namespace,
 			},
 			Labels: map[string]string{
 				CDI_LABEL_KEY:     CDI_LABEL_VALUE,                               //filtered by the podInformer
 				CLONING_LABEL_KEY: CLONING_LABEL_VALUE + "-" + generatedLabelStr, //used by podAffity
+				// this label is used when searching for a pvc's cloner source pod.
+				LabelClonePvc: pvc.Name,
+			},
+			OwnerReferences: []metav1.OwnerReference{
+				metav1.OwnerReference{
+					APIVersion:         "v1",
+					Kind:               "PersistentVolumeClaim",
+					Name:               pvc.Name,
```

Comment:
Owner references assume both the owner and child object are in the same namespace.

We're going to have to work through this situation a bit.  Most likely we'll need to use a finalizer and our own logic to clean up the source pod rather than using the owner reference directly.